runtime/src/types.rs

```diff
@@ -346,6 +349,7 @@ impl Default for Features {
             schedule_control: None,
             key_manager_quote_policy_updates: true,
             key_manager_status_updates: true,
+            key_manager_master_secret_rotation: false,
```

Comment:
I assume this is `false` by default because it only makes sense for key manager runtimes? Otherwise it would make sense to have it default to `true` or introduce a different way of querying for key manager features (e.g. a separate local RPC for that) -- not sure if the complexity is worth it though.

Reply:
> I assume this is `false` by default because it only makes sense for key manager runtimes?

True. As key manager is a runtime, I thought that I could use these features.